### DISCUSSION TITLE
How do you ip ban someone?

### DISCUSSION THREAD

**Discussion by Oo1Toyo1oO** [LBPUnion/ProjectLighthouse#852] [Q&A] [answered]
I never had to but now i do. I know that it's possible through cloudfare but im pretty sure lighthouse also has a build in feature for that? Could someone help me out?

**Comment by sudokoko** [comment, marked as answer]
Lighthouse does not have a built in feature. Beacon uses fail2ban linked with CloudFlare.

**Reply by Oo1Toyo1oO** [reply]
How do you add that?

**Reply by sudokoko** [reply]
Here's a guide you can follow :)
https://niksec.com/using-fail2ban-with-cloudflare/

**Reply by Oo1Toyo1oO** [reply]
Oh jesus thats really complicated.

**Reply by sudokoko** [reply]
Yeahh - you can also just set up a WAF rule within Cloudflare. It's not too hard to do

**Reply by Oo1Toyo1oO** [reply]
What's the difference? And how do you do that?

**Reply by Oo1Toyo1oO** [reply]
Just in the tools section where you can manually type an ip adress out to be blocked? I did set up a rate limiting rule too btw before.

**Reply by sudokoko** [reply]
> What's the difference?

No real difference, works the same.

> Just in the tools section where you can manually type an ip adress out to be blocked?

Yes, I believe that's where it is

**Reply by Oo1Toyo1oO** [reply]
I just realized i did ip ban someone before but that was i got the ip from cloudfare bc they triggered a rule how can you get someones ip from within the server itself?

**Reply by Oo1Toyo1oO** [reply]
Ah, in the game tokens, alr thx.

**Reply by Oo1Toyo1oO** [reply]
Nvm it only showes a few.

**Reply by sudokoko** [reply]
> Ah, in the game tokens, alr thx.

Yeah. They're in the database

**Reply by sudokoko** [reply]
> Nvm it only showes a few.

GameTokens expire after an hour, there's no other way to get a user's IP address from Lighthouse.

**Reply by Oo1Toyo1oO** [reply]
Damn... so after an hour other than them triggering a rule action on cloudfare there's no other way to ip ban after an hour?

**Reply by sudokoko** [reply]
You could probably set up some sort of rule in cloudflare that logs requests IPs but personally I've never done that before, but yeah. From a Lighthouse perspective you can't

**Reply by Oo1Toyo1oO** [reply]
@sudokoko what's the difference between block from this website zone and block all websites in account?

**Reply by sudokoko** [reply]
You'd want to do this website zone most likely. Doing all websites would IP ban them from all of the sites you have registered under your Cloudflare account.

**Reply by Oo1Toyo1oO** [reply]
Alright, thanks!
